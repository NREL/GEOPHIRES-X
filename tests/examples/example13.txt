----------------------------------------------------------------------------------------
GEOPHIRES v3.0 Input File
Example 13
This example file tests redrilling due to excessive drawdown and use of cogen bottoming cycle as end-use
Created by Koenraad Beckers on 11/20/2023
----------------------------------------------------------------------------------------

*** Subsurface technical parameters ***
****************************************
Reservoir Model,4,			--- Percentage thermal drawdown model
Drawdown Parameter,0.01, 		--- In [1/year].
Reservoir Depth,4, 			--- [km]
Number of Segments,1,			--- [-]
Gradient 1,50, 				--- [deg.C/km]
Number of Production Wells,2,		--- [-]
Number of Injection Wells,2,		--- [-]
Production Well Diameter,6,		--- [inch]
Injection Well Diameter,6,		--- [inch]
Ramey Production Wellbore Model,0,	--- Should be 0 (disable) or 1 (enable)
Production Wellbore Temperature Drop,3,	--- [deg.C]
Injection Wellbore Temperature Gain,0,	--- [deg.C]
Production Flow Rate per Well,50, 	--- [kg/s]
Maximum Temperature,400,		--- [deg.C]
Reservoir Volume Option,4,		--- Should be 1, 2, 3, or 4. See manual for details.
Reservoir Volume,1.25e8,		--- [m3] (required for reservoir volume option 3 and 4
Water Loss Fraction,0.0,		--- [-] (total geofluid lost)/(total geofluid produced)
Injectivity Index,10,			--- [kg/s/bar]
Productivity Index,10,			--- [kg/s/bar]
Injection Temperature,50,		--- [deg.C]
Maximum Drawdown,0.12,			--- [-] Maximum allowable drawdown before redrilling (a value of 0.1 means redrilling of 10% drawdown)
Reservoir Heat Capacity,1000,		--- [J/kg/K]
Reservoir Density,2700,			--- [kg/m3]
Reservoir Thermal Conductivity,3,	--- [W/m/K]


*** Surface technical parameters ***
************************************
End-Use Option,42,			--- [-] Co-gen bottoming cycle with heat the main product
Circulation Pump Efficiency,0.8,	--- [-]
Utilization Factor,0.8,			--- [-]
Surface Temperature,12,			--- [deg.C]
Ambient Temperature,12,			--- [deg.C]
End-Use Efficiency Factor,0.8,		--- [-]

District Heating Demand Option,1,					--- Should be 1 or 2. See manual or below for option details
District Heating Demand File Name,Examples/cornell_heat_demand.csv,		--- hourly MW thermal demand in a CSV file
District Heating Demand Data Time Resolution,1,				--- 1 for hourly, 2 for daily
District Heating Demand Data Column Number,2,				---

Peaking Fuel Cost Rate,0.0273, 					--- [$/kWh] Cost of natural gas for peak boiler use
Peaking Boiler Efficiency,0.85,			`		--- Should be between 0 and 1, defaults to 0.85
District Heating Piping Cost Rate,1200,					--- [$/m] used for calculating surface piping capital cost for district heat
District Heating Road Length,3, 				---[km] supersedes model option 2 if any value is entered


*** Economic/Financial Parameters ***
*************************************
Plant Lifetime,30,						--- [years]
Economic Model,2,						--- Should be 1 (FCR model), 2 (Standard LCOE/LCOH model), or 3 (Bicycle model).
Discount Rate,0.05,
Inflation Rate During Construction,0,				--- [-]
Well Drilling and Completion Capital Cost Adjustment Factor,1,	--- [-] Use built-in well cost correlation as is
Well Drilling Cost Correlation,1,				--- [-] Use built-in well drilling cost correlation #1
Reservoir Stimulation Capital Cost,1,				--- [M$/injection well] Reservoir stimulation capital cost per injection well
Surface Plant Capital Cost Adjustment Factor,1,			--- [-] Use built-in surface plant cost correlation as is
Field Gathering System Capital Cost Adjustment Factor,1,	--- [-] Use built-in pipeline distribution cost correlation as is
Exploration Capital Cost Adjustment Factor,0,			--- [-] Use built-in exploration cost correlation as is
Wellfield O&M Cost Adjustment Factor,1,				--- [-] Use built-in wellfield O&M cost correlation as is
Water Cost Adjustment Factor,1,					--- [-] Use built-in water cost correlation as is
Surface Plant O&M Cost Adjustment Factor,1,			--- [-] Use built-in surface plant O&M cost correlation as is


*** Simulation Parameters ***
Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
Time steps per year,3,			--- [1/year]
