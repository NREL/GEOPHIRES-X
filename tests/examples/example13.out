                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.2.11
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-01-18
 Simulation Time:  10:31
 Calculation Time:      0.013 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Cogeneration Bottoming Cycle, Electricity sales considered as extra income
      Average Net Electricity Production:                     3.16 MW
      Average Direct-Use Heat Production:                    15.98 MW
      Direct-Use heat breakeven price:                       13.08 USD/MMBTU
      Number of production wells:                             2
      Number of injection wells:                              2
      Flowrate per production well:                          50.0 kg/sec
      Well depth (or total length, if not vertical):          4.0 kilometer
      Geothermal gradient:                                    0.0500 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model = Standard Levelized Cost
      Interest Rate:                                          5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       80.0 %
      Project NPV:                                         -43.90 MUSD
      Project IRR:                                          -0.06 %
      Project VIR=PI=PIR:                                    0.23
      Project MOIC:                                         -0.21

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             2
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          4.0 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 50.0 degC
      User-provided production well temperature drop
      Constant production well temperature drop:              3.0 degC
      Flowrate per production well:                          50.0 kg/sec
      Injection well casing ID:                               6 in
      Production well casing ID:                              6 in
      Number of times redrilling:                             1
      Power plant type:                                       Subcritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0500 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Annual Percentage Thermal Drawdown Model
      Annual Thermal Drawdown:                                1.000 1/year
      Bottom-hole temperature:                              212.00 degC
      Warning: the reservoir dimensions and thermo-physical properties
               listed below are default values if not provided by the user.
               They are only used for calculating remaining heat content.
      Reservoir volume provided as input
      Reservoir volume:                               125000000 m**3
      Reservoir hydrostatic pressure:                       100.00 kPa
      Plant outlet pressure:                               2261.62 kPa
      Production wellhead pressure:                        2330.57 kPa
      Productivity Index:                                    10.00 kg/sec/bar
      Injectivity Index:                                     10.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir heat capacity:                             1000.00 J/kg/kelvin


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       209.0 degC
      Average Production Temperature:                       197.0 degC
      Minimum Production Temperature:                       184.4 degC
      Initial Production Temperature:                       209.0 degC
      Average Reservoir Heat Extraction:                     62.49 MW
      Wellbore Heat Transmission Model = Constant Temperature Drop:       3.0 degC
      Average Injection Well Pump Pressure Drop:          -1407.4 kPa
      Average Production Well Pump Pressure Drop:          1635.9 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      33.28 MUSD
         Drilling and completion costs per well:              8.32 MUSD
         Stimulation costs:                                   1.00 MUSD
         Surface power plant costs:                          20.07 MUSD
         Field gathering system costs:                        2.32 MUSD
         Total surface equipment costs:                      22.39 MUSD
         Exploration costs:                                   0.00 MUSD
      Total capital costs:                                   56.67 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.69 MUSD/yr
         Power plant maintenance costs:                       1.32 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
      Total operating and maintenance costs:                  3.15 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.11 MW/(kg/s)
      Maximum Total Electricity Generation:                   3.40 MW
      Average Total Electricity Generation:                   3.40 MW
      Minimum Total Electricity Generation:                   3.40 MW
      Initial Total Electricity Generation:                   3.40 MW
      Maximum Net Electricity Generation:                     3.24 MW
      Average Net Electricity Generation:                     3.16 MW
      Minimum Net Electricity Generation:                     3.09 MW
      Initial Net Electricity Generation:                     3.24 MW
      Average Annual Total Electricity Generation:           23.57 GWh
      Average Annual Net Electricity Generation:             21.93 GWh
      Initial pumping power/net installed power:            100.00 %
      Maximum Net Heat Production:                           20.07 MW
      Average Net Heat Production:                           15.98 MW
      Minimum Net Heat Production:                           11.71 MW
      Initial Net Heat Production:                           20.07 MW
      Average Annual Heat Production:                       110.74 GWh
      Average Pumping Power:                                  0.24 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR     THERMAL             GEOFLUID             PUMP             NET              NET             FIRST LAW
           DRAWDOWN           TEMPERATURE           POWER           POWER             HEAT            EFFICIENCY
                                (deg C)             (MW)            (MW)              (MW)               (%)
   0         1.0000              209.00             0.1589          3.2429             20.0682               7.6272
   1         0.9922              207.36             0.1701          3.2317             19.5110               7.6008
   2         0.9843              205.72             0.1811          3.2206             18.9538               7.5748
   3         0.9765              204.09             0.1921          3.2097             18.3965               7.5490
   4         0.9686              202.45             0.2029          3.1988             17.8393               7.5236
   5         0.9608              200.81             0.2136          3.1882             17.2821               7.4985
   6         0.9530              199.17             0.2241          3.1776             16.7249               7.4736
   7         0.9451              197.53             0.2346          3.1672             16.1677               7.4491
   8         0.9373              195.89             0.2449          3.1569             15.6105               7.4249
   9         0.9295              194.26             0.2551          3.1467             15.0532               7.4009
  10         0.9216              192.62             0.2651          3.1366             14.4960               7.3773
  11         0.9138              190.98             0.2751          3.1267             13.9388               7.3539
  12         0.9059              189.34             0.2849          3.1169             13.3816               7.3308
  13         0.8981              187.70             0.2946          3.1072             12.8244               7.3080
  14         0.8903              186.07             0.3042          3.0976             12.2672               7.2854
  15         0.8824              184.43             0.3136          3.0881             11.7099               7.2632
  16         0.9948              207.91             0.1663          3.2354             19.6967               7.6096
  17         0.9869              206.27             0.1775          3.2243             19.1395               7.5834
  18         0.9791              204.63             0.1885          3.2133             18.5823               7.5576
  19         0.9713              202.99             0.1993          3.2024             18.0251               7.5321
  20         0.9634              201.36             0.2100          3.1917             17.4678               7.5068
  21         0.9556              199.72             0.2206          3.1811             16.9106               7.4819
  22         0.9477              198.08             0.2311          3.1706             16.3534               7.4573
  23         0.9399              196.44             0.2415          3.1603             15.7962               7.4329
  24         0.9321              194.80             0.2517          3.1501             15.2390               7.4089
  25         0.9242              193.16             0.2618          3.1400             14.6818               7.3851
  26         0.9164              191.53             0.2718          3.1300             14.1246               7.3616
  27         0.9086              189.89             0.2816          3.1201             13.5673               7.3385
  28         0.9007              188.25             0.2914          3.1104             13.0101               7.3155
  29         0.8929              186.61             0.3010          3.1008             12.4529               7.2929


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             HEAT                 ELECTRICITY                HEAT              RESERVOIR        PERCENTAGE OF
                  PROVIDED               PROVIDED                EXTRACTED          HEAT CONTENT    TOTAL HEAT MINED
                 (GWh/year)             (GWh/year)               (GWh/year)          (10^15 J)           (%)
   1               138.7                   22.7                    471.32               52.98                 3.10
   2               134.8                   22.6                    466.44               51.30                 6.17
   3               130.9                   22.5                    461.56               49.64                 9.21
   4               127.0                   22.5                    456.67               47.99                12.22
   5               123.1                   22.4                    451.79               46.37                15.20
   6               119.2                   22.3                    446.91               44.76                18.14
   7               115.3                   22.2                    442.03               43.17                21.05
   8               111.4                   22.2                    437.15               41.59                23.93
   9               107.4                   22.1                    432.27               40.04                26.77
  10               103.5                   22.0                    427.39               38.50                29.59
  11                99.6                   21.9                    422.51               36.98                32.37
  12                95.7                   21.9                    417.63               35.47                35.12
  13                91.8                   21.8                    412.74               33.99                37.84
  14                87.9                   21.7                    407.86               32.52                40.52
  15                84.0                   21.7                    402.98               31.07                43.18
  16               130.0                   22.5                    460.47               29.41                46.21
  17               136.1                   22.6                    468.06               27.73                49.29
  18               132.2                   22.6                    463.18               26.06                52.34
  19               128.3                   22.5                    458.30               24.41                55.36
  20               124.4                   22.4                    453.42               22.78                58.34
  21               120.5                   22.3                    448.54               21.16                61.30
  22               116.6                   22.3                    443.66               19.56                64.22
  23               112.7                   22.2                    438.78               17.99                67.11
  24               108.7                   22.1                    433.90               16.42                69.96
  25               104.8                   22.0                    429.01               14.88                72.79
  26               100.9                   22.0                    424.13               13.35                75.58
  27                97.0                   21.9                    419.25               11.84                78.34
  28                93.1                   21.8                    414.37               10.35                81.07
  29                89.2                   21.8                    409.49                8.88                83.77
  30                57.3                   14.5                    270.28                7.90                85.54
